frontend/src/metabase/lib/expressions/config.js

```diff
@@ -226,12 +226,22 @@ export const MBQL_CLAUSES = {
     type: "boolean",
     args: ["expression", "expression", "expression"],
   },
+  interval: {
+    displayName: "timeInterval",
```

Comment:
Unfortunately, the new function was called `interval` when there was existing `time-interval` function. It does a different thing, but anyway.

Reply:
Should we call this function `timeSpan` maybe to avoid confusion because of similarities in naming with the existing one?

Reply:
I like `timespan` (or `timeSpan`). Should we get an approve from someone?

Reply:
`timeSpan` works for me.

Reply:
Changed this to `timeSpan`. Also changed `relativeDatetime` to `relativeDateTime` to match this.